modules/social_features/social_comment/config/install/views.view.comment.yml

```diff
@@ -1,11 +1,17 @@
 langcode: en
 status: true
 dependencies:
+  config:
+    - field.storage.profile.field_profile_first_name
+    - field.storage.profile.field_profile_last_name
+    - field.storage.profile.field_profile_nick_name
   module:
     - comment
-    - node
     - group
+    - node
+    - profile
     - social_comment
+    - social_group
```

Comment:
@VBaldych Why do we need dependency from `social_group` here?
Maybe this is a reason why install script is failing.
Try to remove it and check if it fixes the issue.